mu-plugins/global-fonts/helper-functions.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/**
+ * Specify a font to be preloaded.
+ *
+ * @param array|string $font_faces The font(s) to preload.
+ * @return bool If the font will be preloaded.
+ */
+function global_fonts_preload( $font_faces ) {
```

Comment:
Should this get a `wporg_` prefix? It threw me for a second because I expected it to either have a prefix or be namespaced.

Reply:
Probably, but, if dropping `is_callable()` might as well just drop these global functions, and force dependencies to call the namespaced function names, which can be `use function` imported for simpler code.